x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -181,9 +180,6 @@ private static Object valueMapper(CsvTestUtils.Type type, Object value) {
         }
         if (type == CsvTestUtils.Type.GEO_POINT || type == CsvTestUtils.Type.CARTESIAN_POINT) {
             // Point tests are failing in clustered integration tests because of tiny precision differences at very small scales
-            if (value instanceof GeoPoint point) {
-                return normalizedGeoPoint(point.getX(), point.getY());
-            }
             if (value instanceof String wkt) {
```

Comment:
I would remove this check and do the casting directly. I prefer to have a ClassCastException in the test that hide that error and fail somewhere else.